client/my-sites/plans-features-main/hooks/experiments/use-longer-plan-term-default-experiment.ts

```diff
@@ -0,0 +1,47 @@
+import { useExperiment } from 'calypso/lib/explat';
+
+const useTermFromExperimentVariant = (
```

Comment:
Not sure if related to what I'm seeing with the control variant issue in testing https://github.com/Automattic/wp-calypso/pull/96174#issuecomment-2492619164, but do we need a case for treatment 1 `emphasize_savings_only` here?

Reply:
Technically, it wouldn't make a difference, since the term/interval is defaulted to yearly if we don't pass one from here. The eligibility for term savings is then set based on variation not being control. So that should be fine too. 

I'm running tests to verify.

Reply:
Ok I think I got it sorted. Apparently `variationName` comes back as `null` for control. So needed to update `isEligibleForTermSavings` below:

```
	isEligibleForTermSavings: !! (
		experimentAssignment?.variationName && experimentAssignment.variationName !== 'control'
	)
```

Should test fine now :-)